core/websock.js

```diff
@@ -22,9 +22,11 @@ import * as Log from './util/logging.js';
 const ENABLE_COPYWITHIN = false;
 const MAX_RQ_GROW_SIZE = 40 * 1024 * 1024;  // 40 MiB
 
-export default class Websock {
+export default class WebChannel {
     constructor() {
-        this._websocket = null;  // WebSocket object
+        this._webChannel = null;     // WebSocket or RTCDataChannel object
```

Comment:
Now that we have our own name for the wrapper I think we should start clearing up the naming. So can we come up with something here that doesn't copy the name of the wrapper?

Reply:
any ideas? naming things is hard >:)

Reply:
JavaScript fortunately has a tendency for long names, so we don't have to be too afraid of that. Perhaps `_underlyingSocket`, `_rawChannel` or something along those lines?